cmd/format.go

```diff
@@ -0,0 +1,46 @@
+package cmd
+
+import (
+	"os"
+	"strings"
+
+	"github.com/kartverket/skipctl/pkg/constants"
+	"github.com/kartverket/skipctl/pkg/manifest"
+	"github.com/spf13/cobra"
+)
+
+var formatCmd = &cobra.Command{
+	Use:   "format",
+	Short: "Formats .jsonnet files",
+	Long:  "Recursively formats all .jsonnet files in the specified path",
```

Comment:
Lag beskrivelsen generisk, du kan eventuelt liste opp støttede formater. Understrek også at formatteringen vil endre filene in-place.